src/StackAdmin/Azs.Storage.Admin/Module/Azs.Storage.Admin/Azs.Storage.Admin.psd1

```diff
@@ -8,140 +8,125 @@ Licensed under the MIT License. See License.txt in the project root for license
-    CmdletsToExport   = @()
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @()
 
-    # Variables to export from this module
-    # VariablesToExport = @()
+# Variables to export from this module
+# VariablesToExport = @()
 
-    # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-    AliasesToExport   = @()
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = @()
 
-    # DSC resources to export from this module
-    # DscResourcesToExport = @()
+# DSC resources to export from this module
+# DscResourcesToExport = @()
 
-    # List of all modules packaged with this module
-    # ModuleList = @()
+# List of all modules packaged with this module
+# ModuleList = @()
 
-    # List of all files packaged with this module
-    # FileList = @()
+# List of all files packaged with this module
+# FileList = @()
 
-    # Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
-    PrivateData       = @{
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
 
-        PSData = @{
+    PSData = @{
 
-            # Tags applied to this module. These help with module discovery in online galleries.
-            Tags         = @('AzureStack', 'Admin', 'Storage', 'SRP')
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = @('AzureStack', 'Admin', 'Storage', 'SRP')
 
-            # A URL to the license for this module.
-            LicenseUri   = 'https://aka.ms/azps-license'
+        # A URL to the license for this module.
+        LicenseUri   = 'https://aka.ms/azps-license'
 
-            # A URL to the main website for this project.
-            ProjectUri   = 'https://github.com/Azure/azure-powershell'
+        # A URL to the main website for this project.
+        ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
-            # A URL to an icon representing this module.
-            # IconUri = ''
+        # A URL to an icon representing this module.
+        # IconUri = ''
 
-            # ReleaseNotes of this module
-            ReleaseNotes = '
-            * Module dependencies updated
-                * AzureRM.Resources
-            '
+        # ReleaseNotes of this module
+        ReleaseNotes = '* SDK for AzureStack SRP in 2019-08-08-preview'
```

Comment:
Could you please list the Breaking Changes in the release notes. Please provide detailed information like Old Usage Vs New Usage

Reply:
Updated